amethyst_renderer/src/lib.rs

```diff
@@ -65,6 +65,11 @@
 #![deny(missing_docs)]
 #![doc(html_logo_url = "https://tinyurl.com/jtmm43a")]
 
+// for error-chain
+#![recursion_limit="4096"]
```

Comment:
Why have 4096 as the limit?
error_chain recommends 1024, and if I remember correctly, newer rust versions have this limit already set to 1024.